frontend/catalyst/qfunc.py

```diff
@@ -256,15 +264,21 @@ def processing_fn(results):
     single_shot_qnode.device = new_dev
 
     def one_shot_wrapper(*args, **kwargs):
-        def wrap_single_shot_qnode(*_):
+        def wrap_single_shot_qnode(*_, **kwargs):
             return single_shot_qnode(*args, **kwargs)
 
         arg_vmap = jnp.empty((total_shots,), dtype=float)
-        results = catalyst.vmap(wrap_single_shot_qnode)(arg_vmap)
+        results = catalyst.vmap(wrap_single_shot_qnode)(arg_vmap, **kwargs)
```

Comment:
Does including `kwargs` affect vmap in any way?

Besides affecting the vmap call, these two lines seem like a non-functional change to me since `single_shot_qnode` is already invoked with those `kwargs`, so I'm just curious.

Reply:
you are right. I forgot to remove the redundant  `**kwargs`